examples/ai_voltage_hw_timed.py

```diff
@@ -0,0 +1,43 @@
+﻿"""Example of AI voltage sw operation."""
```

Comment:
English has a standard order for using multiple adjectives: https://www.grammarly.com/blog/adjective-order/ . If you don't follow this order, it sounds slightly awkward.

The existing examples don't follow this order and have other style problems like lowercase "sw". 

It would be better to reword the text so that it doesn't contain a long list of adjectives. Consider reusing text from the LabVIEW, ANSI C, CVI, or .NET examples. For example:
```
This example demonstrates how to acquire a finite amount of data
using the DAQ device's internal clock.
```

Reply:
The description is updated by referring LabVIEW examples.